content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/api-filter-attributes.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
```

Comment:
```suggestion
# Search through API calls improved
```
not totally satisfied with this suggestion.. what do you think?

Reply:
Only get what you need from the API?